### DISCUSSION TITLE
[SOLVED] Inav 5, FC H743 and RX Fs-x6b... what's wrong?

### DISCUSSION THREAD

**Discussion by nrlourenzo** [iNavFlight/inav#8430] [Q&A] [answered]
Hi All,

Please see the video - I'm trying to connect a TX16s to a RX FS-X6B (iBUS) and Matek H743 - It works well for some seconds after booting the TX and FC but then if losses connectivity to the TX. Any clue? 

Thanks

[https://youtube.com/shorts/wHvojrDOEQE](url)

**Comment by nrlourenzo** [comment]
I was advised to move the TX from RX since it was breaking the connection and it seemed to work.. but still rudder servo (aileron ch4) is not behaving as expected:

[https://youtu.be/aZGCR0oBdpQ](url)

**Comment by breadoven** [comment, marked as answer]
How is the RX connected to the FC and what settings are used for the receiver ?

**Reply by nrlourenzo** [reply]
Found the problem!

[https://photos.app.goo.gl/GocA9DCGj3iwELv69](url)

[https://photos.app.goo.gl/jG1eE2oc6vT5ZBxk7](url)
